### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by FranDRoss**

### CODE REVIEW

File: staff/francesc-destruels/precourse/calculator2/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Calculator
+
+It's a basic bowser calculator.
```
Comment (didactor91) [inline staff/francesc-destruels/precourse/calculator2/README.md:3]: very small description!

Comment (FranDRoss) [reply staff/francesc-destruels/precourse/calculator2/README.md:3]: acosador xD

File: staff/francesc-destruels/precourse/calculator2/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+// Script para cazar los numeros y lo operadores por teclado;
+document.onkeypress = function(){
+    let key = String.fromCharCode(event.keyCode);
+    let keyCheck = event.keyCode;
+    let numbers = ["0","1","2","3","4","5","6","7","8","9"]
```
Comment (llorenscm) [inline staff/francesc-destruels/precourse/calculator2/index.js]: @FranDRoss falta ;

File: staff/francesc-destruels/precourse/calculator2/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+body {
+    font-family: arial, helvetica, serif;
+    background-color: aqua;
+}
+
+
+
+#calculator {
+	background-color: #190B07;
```
Comment (llorenscm) [inline staff/francesc-destruels/precourse/calculator2/style.css:11]: @FranDRoss espacio de mas

File: staff/francesc-destruels/precourse/bingo/index.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
+// Crea un array con todos los numeros de 1 al 90 ordenados aleatoriamente. Cada turno se mira el siguiente.
+function generateNumberByTurn(){
+    allNumbers = [];
+    while (allNumbers.length < 90){
+        if (allNumbers.includes(number = Math.floor(Math.random()*(91-1)+1)) === false){
+                allNumbers.push(number);
+        }
+    }
+} 
+
```
Comment (daniscript) [inline staff/francesc-destruels/precourse/bingo/index.js:16]: You coud use the shuffle [Fisher Yates method](https://bost.ocks.org/mike/shuffle/), you would spend less resouces to get an array random numbers.

Comment (FranDRoss) [reply staff/francesc-destruels/precourse/bingo/index.js:16]: I would look it, but it will be impossible to apply it in less than a couple weeks.

**Pull request merged**